### PR TITLE
Feature/GDEV-109

### PR DESCRIPTION
**Pull request by KerstenBreuer**
Implements a simple UI for requesting access to a specific dataset.

### CODE REVIEW

File: src/backendCalls/metadata.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { sleep } from '../utils/funcUtils';
+import datasets from '../data/datasets';
+import { dataset, datasetOutline } from '../dataModels/datasets';
+
+export const getDsMetadata: (dsId: string) => dataset = (dsId) => {
+    // currently, looks up in the hardcoded datasets object;
+    // later, will get the info from a backend API
+    
+    sleep(2);
```
Comment (ac-jorellanaf) [inline src/backendCalls/metadata.ts:9]: Why are we sleeping before getting the metadata?

File: src/backendCalls/requests.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { sleep } from '../utils/funcUtils';
+import requests from '../data/requests';
+import { request, requestOutline} from '../dataModels/requests'
+
+export const getRequestList: () => requestOutline[] = () => {
+    // currently, looks up in the hardcoded requests object;
+    // later, will get the info from a backend API
+    sleep(2)
```
Comment (ac-jorellanaf) [inline src/backendCalls/requests.ts:8]: Same question about sleep. What's it doing?

File: src/components/Explore/RequestAccess.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    return (
+        <div>
+            {wasExpanded ? (
+                    wasSend ? (
+                            <div 
+                                className="w3-panel w3-round-xlarge w3-center w3-green w3-text-black"
+                                style={{margin: "16px"}}
+                            >
+                                Request was send.
```
Comment (ac-jorellanaf) [inline src/components/Explore/RequestAccess.tsx:16]: sent

File: src/components/Request/RequestContent.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                                <br/>
+                                {reqMetadata.history
+                                    .sort((a,b) => (a.datetime > b.datetime) ? 1 : -1)
+                                    .map( (event) => (
+                                        <div 
+                                            className="w3-panel"
+                                            key={event.datetime.toUTCString()}
+                                        >
+                                            <span className="w3-text-green w3-small">
+                                                {event.datetime.getHours()}:
```
Comment (ac-jorellanaf) [inline src/components/Request/RequestContent.tsx:54]: Can't this be serialised into a single call?

**Review by ac-jorellanaf — APPROVED** [review]

File: src/backendCalls/metadata.ts (hunk 1 above)
Comment (KerstenBreuer) [reply src/backendCalls/metadata.ts:9]: This is just to simulate some waiting time to get the results from backend.

File: src/components/Request/RequestContent.tsx (hunk 4 above)
Comment (KerstenBreuer) [reply src/components/Request/RequestContent.tsx:54]: You're right. Thanks

**Pull request merged**